model-optimizer/extensions/ops/elementwise.py

```diff
@@ -205,9 +206,17 @@ class Minimum(Elementwise):
 class Round(Elementwise):
     enabled = False
```

Comment:
Add attribute `enabled` in the base class for `UnaryElementwise` and `Elementwise` and refactor all operations in this file.

Reply:
Done.